src/route.ts

```diff
@@ -66,6 +66,7 @@ function routeParams(root: string, cwd: string, parts: string[], exts: string[])
         const path = join(root, cwd, first + ext);
         if (existsSync(path)) {
           if (!statSync(path).isFile()) return; // ignore non-files
+          if (first.match(/\[.*\]/)) continue; // skip [param]
```

Comment:
This should use the stricter regex we use elsewhere for params. Let me see if I can fix…

Reply:
Also, can we pull this test out of the inner loop?